recipes/pyblake2/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = '1.1.0' %}
+{% set sha256 = '3a850036bf42053c74bfc52c063323ca78e40ba1f326b01777da5750a143631a' %}
+{% set name = 'pyblake2' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+build:
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
```

Comment:
Feel free to keep or drop the `py34` as you like.

Reply:
Ah sorry. This was confusing. We should drop the `features` altogether as this is `python` and so already tracks `vc` that way.